File: python/orang.py
class orang:
    def __init__(self, no_ktp="", nama="", alamat=""):
        self.no_ktp = no_ktp
        self.nama = nama
        self.alamat = alamat

    # Getter methods
    def get_no_ktp(self):
        return self.no_ktp

    def get_nama(self):
        return self.nama

    def get_alamat(self):
        return self.alamat

    # Setter methods
    def set_no_ktp(self, no_ktp):
        self.no_ktp = no_ktp

    def set_nama(self, nama):
        self.nama = nama

    def set_alamat(self, alamat):
        self.alamat = alamat

File: cpp/orang.cpp
#pragma once
#include <bits/stdc++.h>
#include <iostream>
#include <string>

using namespace std;

class orang{
    private:
        string noKTP;
        string nama;
        string alamat;

    // CONSTRUCTOR
    public:
        orang(){
            this->noKTP ="";
            this->nama = "";
            this->alamat = "";
        }

        orang(string noKTP, string nama, string alamat){
            this->noKTP = noKTP;
            this->nama = nama;
            this->alamat = alamat;
        }

        // DESTRUCTOR
        ~orang(){
        }

    // GETTER
    string get_noKTP(){
        return noKTP; 
    }

    string get_nama(){
        return nama; 
    }

    string get_alamat(){
        return alamat; 
    }

    // SETTER
    void set_noKTP(string noKTP){
        this->noKTP = noKTP;
    }

    void set_nama(string nama){
        this->nama = nama;
    }

    void set_alamat(string alamat){
        this->alamat = alamat;
    }
};
